Env config refactor

hschiau opened the pull request.

hschiau committed:
refactor

hschiau committed:
move constants to env

hschiau committed:
update readme

lcswillems commented on README.md:
```diff
@@ -1,13 +1,19 @@
 ## Configuration
 
-The application expects a wallet (by default `signer.pem`) file to exist a the root level
+After running `npm run create-env-file`, you will need to updade the values inside.
```
Inside what? Inside `.env` file I guess?

lcswillems commented on README.md:
```diff
@@ -1,13 +1,19 @@
-You can update the path for this inside `generateResponses.js`
+The default wallet path is `./signer_wallet.json`. Provide the file or ensure the file exists at the provided path before running the app.
+
+## Disclaimer
```
Let's put the disclaimer before `## Quick start`.

And let's make the text even more warning:

```
This app is for illustration purposes only and is not suited for production environments as it lacks validation and sanitization of the endpoint parameters.
```

lcswillems commented on README.md:
```diff
@@ -1,13 +1,19 @@
-1. Run `npm install` in the project directory
-2. Run `npm start` to start the app on port 3000 (this can be changed in `server.js`)
+1. Run `npm install` to install the app.
+2. Run `npm run create-env-file` to generate a `.env` configuration file. 
```
We could also not add the script and just tell to copy `.env.example` into `.env` and to fill the variables inside. Might remove some magic.

lcswillems commented on server.js:
```diff
@@ -1,38 +1,101 @@
 
-// Start the server
-app.listen(PORT, () => {
+(async () => {
```
Maybe directly at the top of the file:

```
import { config } from "dotenv";

config();
```

And hence there won't be a need anymore to put the code inside an async wrapper function.

hschiau committed:
updates after review

- remove 'create-env-file' script from package.json
- update README
- remove async wrapper function
